ipaserver/plugins/baseruserfas.py

```diff
@@ -86,6 +87,13 @@
         maxlength=255,
         normalizer=lambda value: value.strip(),
     ),
+    Str(
```

Comment:
I think we need a custom validator for this type of field to verify that it's an actual URL with ``http`` or ``https`` protocol.

Reply:
done, added a simple validator using urlparse